app/services/doc_auth/lexis_nexis/requests/true_id_request.rb

```diff
@@ -57,9 +63,13 @@ def password
 
         def workflow
           if acuant_sdk_source?
-            config.trueid_noliveness_nocropping_workflow
+            include_liveness? ?
+              config.trueid_liveness_nocropping_workflow :
+              config.trueid_noliveness_nocropping_workflow
           else
-            config.trueid_noliveness_cropping_workflow
+            include_liveness? ?
+              config.trueid_liveness_cropping_workflow :
+              config.trueid_noliveness_cropping_workflow
```

Comment:
if these are the same either way, do we need an `if` statement?

Reply:
@night-jellyfish , they are different one is liveness, the other is noliveness.

Reply:
Oh I see - I noticed that each case was checking for liveness, but I didn't notice that each case's default was different for cropping. Thanks!